sdk/textanalytics/ai-text-analytics/src/lro/analyze/poller.ts

```diff
@@ -64,7 +63,6 @@ export class BeginAnalyzeBatchActionsPoller extends AnalysisPoller<
       {
         requestOptions,
         tracingOptions,
-        displayName,
```

Comment:
curious, what happened to `displayName`?

Reply:
I do not remember! I added it back here: https://github.com/Azure/azure-sdk-for-js/pull/14162.